spec/lib/slack_adapter_spec.rb

```diff
@@ -0,0 +1,132 @@
+require 'rails_helper'
+
+SlackAdapter.team_id = "12345"
+SlackAdapter.token = "cool-token"
+SlackAdapter.team_name = "test-team"
+
+describe SlackAdapter do
+  subject { SlackAdapter.new }
+
+  describe "#channels_create" do
+    it "is a thin wrapper around Slack Gem's method" do
+      slack_double = double(channels_create: true)
+      stub_slack!(slack_double)
+      expect(slack_double).to receive(:channels_create)
+      subject.channels_create
+    end
+  end
+
+  describe "#send_team_invite" do
+    before do
+      allow(Time).to receive(:now).and_return(42)
+    end
+
+    it "executes a post with the right queries" do
+      email = "some_email"
+      payload = "_attempts=1&channels=C067NNHEY&email=#{email}&set_active=true&team_id=12345&token=cool-token"
+      proper_args = {
+        method: :post,
+        url: "https://test-team.slack.com/api/users.admin.invite?t=42",
+        payload: payload
+      }
+
+      expect(RestClient::Request).to receive(:execute).with(proper_args).
+        and_return({ok: true}.to_json)
+      subject.send_team_invite(email)
+    end
+
+    it "returns proper error if applicable" do
+      allow(RestClient::Request).to receive(:execute).
+        and_return("{\"error\":\"already_in_team\"}")
+
+      expect{subject.send_team_invite("email")}.
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.